todo_app/app.py

```diff
@@ -41,21 +43,33 @@ def donelists():
     
     @app.route('/')
     def index():
+        if not github.authorized:
+            return redirect(url_for("github.login"))
         todo_items = todolists()
         doing_items = doinglists()
         done_item = donelists()
         item_view_model = ViewModel(todo_items, doing_items, done_item)
-        return render_template('index.html', view_model = item_view_model)
+        username=session["username"]
+        if username == os.getenv("OAUTHADMIN"):
+        #if username == "smellybeagle":
+            role="ADMIN"
+        else:
+            role="READ ONLY"
```

Comment:
Unimportant, but if we followed the pattern mentioned below, we could update this to just be:
```suggestion
        role = session["role"]
```